numpy/_core/src/umath/string_buffer.h

```diff
@@ -464,8 +652,8 @@ struct call_buffer_member_function {
                 return codepoint_isdigit<enc>(*buf);
             case IMPLEMENTED_UNARY_FUNCTIONS::ISSPACE:
                 return codepoint_isspace<enc>(*buf);
-            case IMPLEMENTED_UNARY_FUNCTIONS::STR_LEN:
```

Comment:
why remove `str_len`? I'm using this for stringdtype I'm pretty sure.

Reply:
I checked this and my understanding is that `str_len` never calls it. It's basically a [similar struct in `stringdtype_ufuncs.cpp`](https://github.com/lysnikolaou/numpy/blob/main/numpy/_core/src/umath/stringdtype_ufuncs.cpp#L1015) that does.